content/kubeone/master/infrastructure/production_recommendations.en.md

```diff
@@ -4,4 +4,61 @@ date = 2020-04-01T12:00:00+02:00
 weight = 4
 +++
 
-TBD
\ No newline at end of file
+## AWS
+
+### ami_id
+
+{{% notice warning %}}
+It's very important to set this terraform variable to avoid future terraform attempts to recreate your control plane
+instance.
+{{% /notice %}}
+
+Can be set AFTER initial `terraform apply`, and you can find its initial discovered values in your terraform state by
```

Comment:
```suggestion
This can be set **after** initial `terraform apply`, and you can find its initial discovered values in your terraform state by
```

I think that we can just bold `after`, it probably doesn't need to be uppercase.